api/ombpdf/views.py

```diff
@@ -0,0 +1,83 @@
+import json
+
+from django.contrib.staticfiles.templatetags.staticfiles import static
+from django.http import FileResponse, Http404, HttpResponse
+from django.shortcuts import render
+from django.urls import reverse
+from django.utils.safestring import SafeString
+
+from ombpdf import html, rawlayout, semhtml
+from ombpdf.document import OMBDocument
+from ombpdf.download_pdfs import ROOT_DIR as DATA_DIR
+from ombpdf.util import get_ltpages
+
+
+def to_urlpath(path):
+    return '/'.join(path.relative_to(DATA_DIR).parts)
+
+
+def get_pdfmap():
+    pdfmap = {}
+    for path in DATA_DIR.glob('**/*.pdf'):
+        pdfmap[to_urlpath(path)] = path
+    return pdfmap
+
+
+_ltpages_cache = {}
+
+
+def to_doc(path):
+    if path not in _ltpages_cache:
+        with path.open('rb') as fp:
+            _ltpages_cache[path] = get_ltpages(fp)
+    return OMBDocument(
+        _ltpages_cache[path],
+        filename=path.relative_to(DATA_DIR),
+    )
+
+
+def index(request):
+    return render(request, 'ombpdf/index.html', {
+        'name': __name__,
```

Comment:
I'm confused by this -- why do we want the module name in our template?

Reply:
Oh lol that's just weird leftover code, I can remove it.  I think it started out as code to just make sure the Flask/Jinja2 template context worked properly, and then it just stuck.